src/edit/InstanceForm.js

```diff
@@ -203,7 +226,14 @@ class InstanceForm extends React.Component {
         </Icon>
       </Button>
     );
-  }
+  };
+
+  isFieldBlocked = fieldName => {
```

Comment:
Please make it a regular method (not an arrow function) as it does not need to be bound.